### PR TITLE
created_models_package

### PR DESCRIPTION
**Pull request by alidurrani1**
created package of models.
structured the project.
added pagination.

### CODE REVIEW

File: app/views.py [hunk 1]
```diff
@@ -90,17 +99,17 @@ def home():
...
-        if check_user != None:
+        if check_user:
             # Storing Session
             session['check_user'] = user_name
 
             # Creating Tokens (Access Token) and (Refresh Token)
 
             session_token = jwt.encode({"user": user_name,
-                                        "exp": datetime.utcnow() + timedelta(seconds=5)},
+                                        "exp": datetime.utcnow() + timedelta(seconds=50)},
```
Comment (maimoonaiqbal94) [inline app/views.py:109]: what is the interval for expiry?

File: models/car_model.py [hunk 2]
```diff
@@ -35,4 +20,4 @@ def get_all(cls):
 
     @classmethod
     def get_by_id(cls, id):
-        return cls.query.get_or_404(id)
+        return cls.query.get_or_404(id)
```
Comment (maimoonaiqbal94) [inline models/car_model.py]: pep8. new line at end of file.

File: app/views.py (hunk 1 above)
Comment (alidurrani1) [reply app/views.py:109]: 50 seconds after the token is created.

File: models/car_model.py (hunk 2 above)
Comment (alidurrani1) [reply models/car_model.py]: resolved.

File: app/views.py [hunk 3]
```diff
@@ -46,10 +45,10 @@ def email_sending(link):
     """
     mail = Mail(app)
     msg = Message(
-            'Hello',
-            sender='alidurrani550@gmail.com',
-            recipients=['anonymouspyf@protonmail.com']
-          )
+        'Hello',
+        sender='alidurrani550@gmail.com',
```
Comment (maimoonaiqbal94) [inline app/views.py:49]: make consts file and use them there.

File: models/user_model.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    username = Column(String(28), nullable=False, unique=True)
+    password = Column(String(28), nullable=False)
+    __tablename__ = 'user'
+
+    def __int__(self, username, password):
+        self.username = username
+        self.password = password
+
+
```
Comment (maimoonaiqbal94) [inline models/user_model.py]: too many extra lines.

File: models/user_model.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+from sqlalchemy import String, Column, Integer
+
+from models.base import Base
+
+# Creating Database
+class User(Base):
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    username = Column(String(28), nullable=False, unique=True)
+    password = Column(String(28), nullable=False)
+    __tablename__ = 'user'
```
Comment (maimoonaiqbal94) [inline models/user_model.py]: table names are named plural in tablenames... it should be **users** here.

File: models/car_model.py [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+from sqlalchemy import String, Column, Integer
+from models.base import Base
+
+class Car(Base):
+    id = Column(String(30), primary_key=True)
+    year = Column(Integer, nullable=False)
+    make = Column(String(28), nullable=False)
+    created_at = Column(String(60), nullable=False)
+    updated_at = Column(String(60), nullable=False)
+    __tablename__ = 'car'
```
Comment (maimoonaiqbal94) [inline models/car_model.py]: cars

File: models/base.py [hunk 7]
```diff
@@ -0,0 +1,3 @@
+from sqlalchemy.orm import declarative_base
+
+Base = declarative_base()
```
Comment (maimoonaiqbal94) [inline models/base.py]: pep8

File: migrations/README [hunk 8]
```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```
Comment (maimoonaiqbal94) [inline migrations/README]: README.md file should be used instead.

File: migrations/versions/5ad0c6d5d9d5_first_migration.py [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+
+# revision identifiers, used by Alembic.
+revision = '5ad0c6d5d9d5'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
```
Comment (maimoonaiqbal94) [inline migrations/versions/5ad0c6d5d9d5_first_migration.py]: i dont think this file is needed

**Review by maimoonaiqbal94 — APPROVED** [review]

**Pull request merged**